src/SFA.DAS.ApprenticeFeedback.Data/Repository/ApprenticeFeedbackRepository.cs

```diff
@@ -19,46 +19,32 @@ public ApprenticeFeedbackRepository(IApprenticeFeedbackDataContext dbContext)
 
         public async Task<Guid?> CreateApprenticeFeedbackTarget(ApprenticeFeedbackTarget feedbackTarget)
         {
-            try
-            {
-                await _dbContext.ApprenticeFeedbackTargets.AddAsync(feedbackTarget);
-                _dbContext.SaveChanges();
-            }
-            catch
-            {
-                return null;
-            }
+            await _dbContext.ApprenticeFeedbackTargets.AddAsync(feedbackTarget);
+            _dbContext.SaveChanges();
```

Comment:
Why does this addasync but then not saveasync?

Reply:
Due to the way the Data Context was interfaced the method wasn't exposed. Have reworked to expose it correctly